src/RNQuery/RNQueryPage.jsx

```diff
@@ -0,0 +1,44 @@
+import {useQuery} from "react-query";
+
+const fetchApi = async() => {
```

Comment:
Can be exported from separate service file instead of global constant

Reply:
created seperate ApiServices.js file